code/game/machinery/jukebox.dm

```diff
@@ -184,10 +185,12 @@ datum/track/New(var/title_name, var/audio)
 		return 1
 
 /obj/machinery/media/jukebox/proc/StopPlaying()
-	var/area/main_area = get_area(src)
-	// Always kill the current sound
-	for(var/mob/living/M in mobs_in_area(main_area))
-		M << sound(null, channel = 4)
+	// var/area/main_area = get_area(src)
```

Comment:
commented code, shouldn't this be uncommented?

Reply:
I don't think this should be uncommented, the tokens handles the stopping of the sound when they get QDELed, making it unnecessary to handle it in other parts

In the Destroy() function, it calls Stop() before going up the parent chain, Stop() takes care of pushing the null sound to the mobs to stop the sound, deregister from the SSProcessing etc